chartos/chartos.yml

```diff
@@ -6,33 +6,74 @@
       on_field: geographic
       cache_duration: 3600
       fields:
-        - "layer.obj_id as id"
-        - "(track_section.data->>'length')::float as length"
-        - "(track_section.data->>'line_code')::int as line_code"
-        - "(track_section.data->>'line_name')::text as line_name"
-        - "(track_section.data->>'track_number')::int as track_number"
-        - "(track_section.data->>'track_name')::text as track_name"
-        - "(track_section.data->>'navigability')::text as navigability"
-        - "(track_section.data->>'slopes')::text as slopes"
-        - "(track_section.data->>'curves')::text as curves"
+        - field_expr: layer.obj_id
+          field_name: id
+          field_type: text
```

Comment:
prefixing everything with field_ probably isn't necessary, but I can live with it